desserts/chokolate desert drink

```diff
@@ -0,0 +1,6 @@
+40g Dark chocolate (70%)
+30g whipped cream
+2 tbsp pepper
```

Comment:
This does not sound too nice

Reply:
Hmm okay, if you don't like pepper, I'll remove it.

> This does not sound too nice